File: formula.java
import java.util.Scanner;

public class QuadraticEquationSolver {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Declaring Variables for input
        double a, b, c;
        // Declaring Variables for processing and output
        double xValue, negativeRoot, positiveRoot;

        // Input coefficients from user
        System.out.print("Enter coefficient A: ");
        a = scanner.nextDouble();
        System.out.print("Enter coefficient B: ");
        b = scanner.nextDouble();
        System.out.print("Enter coefficient C: ");
        c = scanner.nextDouble();

        // Calculate the discriminant and Calculating for Positive and Negative
        xValue = b * b - 4 * a * c;
        negativeRoot = (-b - Math.sqrt(xValue)) / (2 * a);
        positiveRoot = (-b + Math.sqrt(xValue)) / (2 * a);

        // Output the equations
        System.out.println("Negative Root: " + negativeRoot);
        System.out.println("Positive Root: " + positiveRoot);
    }
}

File: Formula.c
#include <stdio.h>
#include <math.h>

int main() {
    // Declaring of Variables for input
    double a, b, c;
    // Declaring of Variable for processing and output
    double xValue, negativeRoot, positiveRoot;

    // Input coefficients from user
    printf("Enter coefficient A: ");
    scanf("%lf", &a);
    printf("Enter coefficient B: ");
    scanf("%lf", &b);
    printf("Enter coefficient C: ");
    scanf("%lf", &c);

    // Calculate the discriminant and Calculating for Positive and Negative
    xValue = b * b - 4 * a * c;
    negativeRoot = (-b - sqrt(xValue)) / (2 * a);
    positiveRoot = (-b + sqrt(xValue)) / (2 * a);

    // Output the equations
    printf("Negative Root: %lf\n", negativeRoot);
    printf("Positive Root: %lf\n", positiveRoot);

    return 0;
}
